packages/css-library/src/stylesheets/base/_fonts.scss

```diff
@@ -0,0 +1,36 @@
+/*
+The uswds-fonts package loads @font-face for:
+  - Roboto Mono Web
+  - Source Sans Pro Web
+  - Merriweather Web
+*/
+@use "uswds-fonts";
```

Comment:
This will add `@font-face` imports for those fonts listed above from the USWDS v3 package. They all include the name "Web" though which is different from v1.